source/common/stats/thread_local_store.cc

```diff
@@ -135,7 +136,8 @@ void ThreadLocalStoreImpl::releaseScopeCrossThread(ScopeImpl* scope) {
   }
 }
 
-std::string ThreadLocalStoreImpl::getTagsForName(const std::string& name, std::vector<Tag>& tags) {
+std::string ThreadLocalStoreImpl::getTagsForName(const std::string& name,
+                                                 std::vector<Tag>& tags) const {
```

Comment:
This whitespace change makes me think you are running the format-fixing script with the clang setup on your machine rather than via docker.  In particular it re-wrapped the line to fit in 80 columns when Envoy standard is 100. You probably want to run the format-fixer like this:
```
./ci/run_envoy_docker.sh './ci/do_ci.sh fix_format'
```

Reply:
i think it is because of const addition the length went up to 106 that is why it wrapped.

Reply:
Sorry.  I stared at that for 1 less second than needed & missed the 'const' addition :)